wlcreate/Start-Asian-Love
# Applying Styling to Pages

## Description
- Updated styling to all pages from April 30 to May 4
    - Color palette of yellow, purple, and red accents.
    - Grids are in 3 columns for consistency.
    - Footer and buttons updated.
    - Home page transformed!

Related Issues: Resolves #3, resolves #5 





Resources:
- https://www.canva.com/
- https://smolcss.dev/
- https://color.adobe.com/explore
- https://fonts.adobe.com/fonts
- https://dev.to/samanthaming/css-variables-8di 
- http://getbem.com/introduction/
- http://www.heropatterns.com/



## Acceptance Criteria

- Styling!
    - Avoid urge to style until the end.
    - Make it work, then make it look good!
    - Instead of styling intermittently, set up empty module.css components to avoid initial errors.
    - Bookmark any inspiration along the way.
    - https://stopaapihate.org/ aesthetic is too severe and harsh.
    - Mood and brand should be welcoming, positive, engaging for all ages.

Acceptance Criteria: 
- Use one of the follow APIs to let the user see news related to anti-Asian hate, Asian racism, and the Stop Asian Hate movement
    - https://newscatcherapi.com/free-news-api
    - https://mediastack.com
- Resource
    - [News API Tutorial](https://youtu.be/xtItzwYG6oQ)
- Optional
    - Pagination
    - Filtering/Sorting capabilities based on date
    - Filtering by location
 - Rallies and Newest Updates (to be added)
    - Provide most current information on rallies and news articles



## Self-check

Please check all that apply:



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="1718" alt="Screen Shot 2021-05-04 at 2 36 45 AM" src="https://user-images.githubusercontent.com/61435324/116969059-a1ca1200-ac83-11eb-8548-56781160116e.png">

<img width="1713" alt="Screen Shot 2021-05-04 at 2 37 02 AM" src="https://user-images.githubusercontent.com/61435324/116969070-a4c50280-ac83-11eb-9551-f07f08e228ca.png">

<img width="1714" alt="Screen Shot 2021-05-04 at 2 37 12 AM" src="https://user-images.githubusercontent.com/61435324/116969089-ae4e6a80-ac83-11eb-9a9f-eb54aef899bd.png">

<img width="1713" alt="Screen Shot 2021-05-04 at 2 37 20 AM" src="https://user-images.githubusercontent.com/61435324/116969105-b6a6a580-ac83-11eb-8c25-b15e1eb0bf0c.png">

<img width="1715" alt="Screen Shot 2021-05-04 at 2 37 31 AM" src="https://user-images.githubusercontent.com/61435324/116969117-bb6b5980-ac83-11eb-9ca3-93a8d534ab49.png">



### After
<img width="1716" alt="Screen Shot 2021-05-04 at 2 34 59 AM" src="https://user-images.githubusercontent.com/61435324/116969151-c1f9d100-ac83-11eb-83de-a9d7f55fb40d.png">

<img width="1715" alt="Screen Shot 2021-05-04 at 2 35 16 AM" src="https://user-images.githubusercontent.com/61435324/116969164-c6be8500-ac83-11eb-8559-7bc5e7dc513c.png">

<img width="1713" alt="Screen Shot 2021-05-04 at 2 35 25 AM" src="https://user-images.githubusercontent.com/61435324/116969168-c920df00-ac83-11eb-9bfa-db8321b94392.png">

<img width="1710" alt="Screen Shot 2021-05-04 at 2 35 37 AM" src="https://user-images.githubusercontent.com/61435324/116969172-caeaa280-ac83-11eb-9ff4-fc9984893743.png">

<img width="1716" alt="Screen Shot 2021-05-04 at 2 35 47 AM" src="https://user-images.githubusercontent.com/61435324/116969189-d2aa4700-ac83-11eb-8cd0-8b8ef29d0d0d.png">

<img width="1713" alt="Screen Shot 2021-05-04 at 2 36 03 AM" src="https://user-images.githubusercontent.com/61435324/116969194-d50ca100-ac83-11eb-97aa-20dfdcb9cbdb.png">



## Testing Steps / QA Criteria
- Switch to global-style branch: `git checkout global-style`
- Git pull if necessary: `git pull origin global-style`
- Launch Start Asian Love: `npm run dev`
- Check each page and corresponding buttons and/or links.
- Check header and footer components and links.
- Resize browser window on each page.
